front/app/services/locale.ts

```diff
@@ -32,21 +32,21 @@
 // imports and definitions
 import { BehaviorSubject, combineLatest } from 'rxjs';
 import { first, map, distinctUntilChanged, filter } from 'rxjs/operators';
-import { includes, isEqual, get } from 'lodash-es';
-import { currentAppConfigurationStream } from 'services/appConfiguration';
+import { includes, get } from 'lodash-es';
+import appConfigurationStream from 'api/app_configuration/appConfigurationStream';
 import { authUserStream } from 'services/auth';
 import { updateUser } from 'services/users';
 import { Locale } from 'typings';
 import { locales } from 'containers/App/constants';
 import { setCookieLocale, getCookieLocale } from 'utils/localeCookie';
 import clHistory from 'utils/cl-router/history';
+import { IAppConfiguration } from 'api/app_configuration/types';
 
 export const LocaleSubject: BehaviorSubject<Locale> = new BehaviorSubject(
   null as any
 );
-const $tenantLocales = currentAppConfigurationStream().observable.pipe(
-  map((tenant) => get(tenant, 'data.attributes.settings.core.locales')),
-  distinctUntilChanged((prev, next) => !isEqual(prev, next))
```

Comment:
What's the reasoning behind removing this operation?

Reply:
It didn't really work with the new currentAppConfigurationStream for some reason and I don't think it's a hugely valuable optimization in any case so I didn't really spend a lot of time debugging it. Once we convert the user, we can hopefully improve this entire setup and not rely on rxjs for it at all.

Reply:
Alright, I wouldn't expect anything to break from this either, but there is a small chance a downstream subscriber has some quirk that it behaves differently without `distinctUntilChanged` upstream. I'm okay with that minimal risk, but if you checked, even better.

Reply:
This particular stream is only used in this file to create the initial setup so it has only one subscriber. We've tested the platform extensively at this point and this doesn't seem to create problems so far but I'll do one final test on staging to make extra sure.

Reply:
Alright, totally fine then :)